### ISSUE TITLE
Fetch data before loading bubble chart

### ISSUE THREAD

**Issue by acrdlph** [convergentcx/Arena#63]
Bubble chart does not display correctly, because of incoming updates (e.g. bubbles move erratically in the beginning + cannot assign different colours to different bubbles)

**Comment by lsaether** [comment]
We should be able to do this by configuring drizzle. I experimented with this a little bit today but couldn't get the configuration right. Ideally we would be able to pass in `Created` event to the initial configuration and then be able to access the events from `this.props.drizzleState`. Alternatively, we may need to implement this as our own Redux store.

**Comment by acrdlph** [comment]
good idea - i have not been able to figure out events with drizzle, but this must be possible, especially since we explicitly pass events in the configuration .. its just not well documented / not much info online
